site/app/libraries/QueueItem.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
Should a few basic unit tests be added for this file?

Reply:
Unit tests might be a little tricky since it is reading the queue files which are constantly being grabbed by the `autograding_workers`, it could be done if we first manually stop the shippers and workers though.

Reply:
Unit tests should not rely on autograding or the website to be functional. Unit tests are run before our e2e tests. Instead you can create very basic json files and call the constructor with that.